.github/workflows/linux-builds-on-master.yaml

```diff
@@ -27,6 +27,11 @@ jobs:
         include:
           - target: x86_64-unknown-linux-gnu
             run_tests: YES
+            snap_arch: amd64
+          - target: aarch64-unknown-linux-gnu # skip-pr
```

Comment:
Should these stay skip-pr?

Reply:
The intent here was PRs only test building the snap, so we check the native arch and one cross arch to make sure I've not fluffed anything.  then master builds more of them because it uploads to the edge channel, and stable uploads all of them to the beta channel.  Does that make sense to you?

Reply:
So master = edge, and stable == releases == beta, and actual stable channel is manual?

Reply:
That is indeed the current plan.